src/pages/docs/deployments/kubernetes/helm-update/index.md

```diff
@@ -115,6 +115,10 @@ Helm deployments using Tar.gz packages can fail if the path is 100+ characters,
 See [https://github.com/OctopusDeploy/Issues/issues/8132](https://github.com/OctopusDeploy/Issues/issues/8132) for more info.  
 :::
 
+:::div{.warning}
+Due to how deployment cancellation currently works, the Helm `--atomic` argument does not result in automatic rollbacks when a deployment is cancelled. Furthermore, if the Octopus deployment timeout is set lower than the Helm timeout, a similar issue may arise. To ensure a smooth deployment experience, we recommend setting a larger Octopus timeout than the Helm timeout.
```

Comment:
nit: If `--atomic` doesn't result in automatic rollback - what does it do? (or is that obvious and I'm just not well versed?)

Also - if the deployment timeout is lower than helm-timeout, what _does_ happen? Or again, is that obvious to people in the know?

Reply:
Added in some more details